src/client.ts

```diff
@@ -108,6 +108,19 @@ export default class Client {
   private async gemMissing(): Promise<boolean> {
     const bundledGems = await this.execInPath("bundle list");
 
+    if (bundledGems.includes("rubocop-lsp")) {
+      vscode.window.showErrorMessage(
+        "The rubocop-lsp gem has been replaced by the ruby-lsp.",
+        {
+          modal: true,
+          detail:
+            "Please, remove the rubocop-lsp and add ruby-lsp to your Gemfile",
```

Comment:
This is an extreme nitpick but I think we should remove the comma after `Please`.

Reply:
I changed it so that it displays the warning and then continues to check for the `ruby-lsp`. If it's not in the `Gemfile`, it'll still prompt people to add it to the bundle.